apps/web/src/components/templates/BlueprintModal.tsx

```diff
@@ -39,6 +56,7 @@ export function BlueprintModal() {
     onSuccess: (template) => {
       localStorage.removeItem('blueprintId');
       if (template) {
+        disableOnboarding();
```

Comment:
Why we are disabling the onboarding? I feel like for the new accounts they won't still know how to do integrations, create the template, use trigger etc.